### PR TITLE
Cloudbank version & properties

### PR DESCRIPTION
**Pull request by andytael**
Updating versions and cleaning up properties

### CODE REVIEW

File: cloudbank-v32/checks/pom.xml [hunk 1]
```diff
@@ -13,9 +13,9 @@
     <version>0.0.1-SNAPSHOT</version>
     <name>checks</name>
     <description>Check Processing Application</description>
-    <properties>
+    <!-- <properties>
```
Comment (markxnelson) [inline cloudbank-v32/checks/pom.xml:16]: did you mean to leave commented out code?

**Review by markxnelson — DISMISSED** [review]

File: cloudbank-v32/checks/pom.xml (hunk 1 above)
Comment (andytael) [reply cloudbank-v32/checks/pom.xml]: Absolutely not :-)

**Review by markxnelson — DISMISSED** [review]

File: cloudbank-v32/README.md [hunk 2]
```diff
@@ -2,7 +2,15 @@
 
 **NOTE:** This document and application is WIP.
 
-To run Cloud Bank you need OBaaS version 1.1.1 [Oracle Backend for Spring Boot and Microservices](https://cloudmarketplace.oracle.com/marketplace/en_US/listing/138899911)
+To run Cloud Bank you need OBaaS version 1.1.3 [Oracle Backend for Spring Boot and Microservices](https://cloudmarketplace.oracle.com/marketplace/en_US/listing/138899911)
+
+## Release notes
+
+1.2.0
```
Comment (markxnelson) [inline cloudbank-v32/README.md:9]: too soon to add 1.2.0?

Comment (andytael) [reply cloudbank-v32/README.md]: Removed rel notes

**Review by markxnelson — APPROVED** [review]

**Review by markxnelson — APPROVED** [review]

**Review by markxnelson — APPROVED** [review]

**Review by markxnelson — DISMISSED** [review]

**Review by markxnelson — APPROVED** [review]

**Review by markxnelson — APPROVED** [review]

**Review by markxnelson — APPROVED** [review]

**Pull request merged**